src/injected/utils/index.js

```diff
@@ -6,16 +6,21 @@ export {
   sendMessage, request, throttle, cache2blobUrl,
 } from '#/common';
 
+const isChrome = !!window.chrome
+  && /^function\s?\w*\s*\(\)\s*{\s*\[native code]\s*}\s*$/.test(
+    (window.chrome.app || {}).getDetails,
+  );
```

Comment:
I suggest to check this in the content script, and pass it to `VMInitInjection`.

Reply:
Can we just use a simple string compare here?

Reply:
We can but I'm afraid Chrome can just change it in the future by adding a space or a newline. I'm not sure all versions of Chrome have the same string.

Reply:
Oh, there's no need for this complicated check if we do it in the content script as you suggested.